standup/status/jinja2/base.html

```diff
@@ -65,70 +65,72 @@
+    </div>
+  </header>
 
-    <div id="main-content" class="container_12 cf">
-      <div id="main-notices" class="grid_12">
-        <div class="notice success">
-          We've switched the way we use Auth0. If you have problems, see
-          <a href="{{ settings.HELP_FAQ_URL }}">our FAQ</a>.
-        </div>
-        {% if messages %}
-          {% for message in messages %}
-            <div class="notice {{ message.tags }}">{{ message }}</div>
-          {% endfor %}
-        {% endif %}
+  <div id="main-content" class="container_12 cf">
+    <div id="main-notices" class="grid_12">
+      <div class="notice success">
+        We've switched the way we use Auth0. If you have problems, see
+        <a href="{{ settings.HELP_FAQ_URL }}">our FAQ</a>.
       </div>
-      {% block before_content %}
-        {% if user.is_authenticated() %}
-          {{ statusize() }}
-        {% endif %}
-      {% endblock %}
-      {% block content %}{% endblock %}
-      {% block after_content %}{% endblock %}
+      {% if messages %}
+        {% for message in messages %}
+          <div class="notice {{ message.tags }}">{{ message }}</div>
+        {% endfor %}
+      {% endif %}
     </div>
+    {% block before_content %}
+      {% if user.is_authenticated() %}
+        {{ statusize() }}
+      {% endif %}
+    {% endblock %}
+    {% block content %}{% endblock %}
+    {% block after_content %}{% endblock %}
+  </div>
 
-    <footer>
-      <div class="container_12 cf">
-        <div class="grid_1">
-          <img src="{{ static('img/logo-grey.png') }}" alt="">
-        </div>
-        <div class="grid_5">
-          Standup is a neat little app that helps to organize and publish
-          asynchronous status updates.
-        </div>
-        <div class="grid_6">
-          <nav>
-            <ul>
-              {% if settings.HELP_FAQ_URL %}
-                <li><a href="{{ settings.HELP_FAQ_URL }}">Help &amp; FAQ</a></li>
-              {% endif %}
-              <li><a href="https://github.com/mozilla/standup/issues">Report a bug</a></li>
-              <li><a href="https://github.com/mozilla/standup">Contribute</a></li>
-            </ul>
-          </nav>
-        </div>
+  <footer>
+    <div class="container_12 cf">
+      <div class="grid_1">
+        <img src="{{ static('img/logo-grey.png') }}" alt="">
       </div>
-    </footer>
-  </div>
+      <div class="grid_5">
+        Standup is a neat little app that helps to organize and publish
+        asynchronous status updates.
+      </div>
+      <div class="grid_6">
+        <nav>
+          <ul>
+            {% if settings.HELP_FAQ_URL %}
+              <li><a href="{{ settings.HELP_FAQ_URL }}">Help &amp; FAQ</a></li>
+            {% endif %}
+            <li><a href="https://github.com/mozilla/standup/issues">Report a bug</a></li>
+            <li><a href="https://github.com/mozilla/standup">Contribute</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+  </footer>
```

Comment:
The rest of the changes in this file are just indentation fixing and removing a spurious `</div>` tag.